src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -664,21 +702,8 @@
             <!-- Strict content model: (h1 p)? (section | article) -->
             <choice>
                 <group>
-                    <a:documentation>single-page HTML.</a:documentation>
-                    <!-- single-page HTML -->
+                    <!-- content document -->
                     <ref name="attrs"/> <!-- @id, @title, @xml:space, @xml:lang, @lang, @dir, @epub:type, @class -->
-                    <element name="header">
-                        <a:documentation>h1</a:documentation>
-                        <ref name="doctitle.headline"/> <!-- h1 -->
-                        <optional>
-                            <a:documentation>p</a:documentation>
-                            <ref name="covertitle"/> <!-- p -->
-                        </optional>
-                        <zeroOrMore>
-                            <a:documentation>p</a:documentation>
-                            <ref name="docauthor"/> <!-- p -->
-                        </zeroOrMore>
-                    </element>
                     <oneOrMore>
```

Comment:
This `<oneOrMore>` should be removed as well. There will always be exactly one top-level section in an EPUB according to the 2020-1 guidelines.